app/controllers/badge_claim_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BadgeClaimController < ApplicationController
+
+  def create
+    @claim = BadgeClaim.new
+  end
+
+  def delete
+    @claim = BadgeClaim.find(params[:claim_id])
+    @claim.delete
```

Comment:
Yeah, I had originally intended to make routes for these endpoints, but ultimately ended up going the admin panel route instead (it made more sense to me, since non-admin users ultimately would never use these endpoints).

Good catch, I didn't mean to leave this code behind. I'll remove it.